app/main.py

```diff
@@ -1,8 +1,25 @@
 class Person:
-    # write your code here
-    pass
+
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        self.people[name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    person_list = []
+    for person in people:
+        name = person.get("name")
+        age = person.get("age")
+        if name is not None and age is not None:
+            person_list.append(Person(name, age))
+    for person in people:
+        if "wife" in person and person["wife"] is not None:
+            wife = Person.people[person["wife"]]
```

Comment:
better to use `get`
fix everywhere 
```suggestion
        if "wife" in person and person.get("wife") is not None:
            wife = Person.people[person["wife"]]
```